assets/scripts/multiple_events_rhai.rhai

```diff
@@ -1,5 +1,15 @@
 fn on_init(name) {
     print(`Hello World! From "${name}" in Init`);
+
+    let parent = world.get_parent(entity);
+    switch type_of(parent) {
```

Comment:
is it possible to do something like:
```rhai
if parent {
  ... access parent
} else {
  ... throw error
}
```
Instead? As in will rhai automatically convert those to truthy values here ?

Reply:
almost, you can check it like this
```rhai
let parent = world.get_parent(entity);
if parent == () {
    print(`Parent doesn't exist`);
} else {
    print(`Parent exists`);
}
```

I have updated the example accordingly